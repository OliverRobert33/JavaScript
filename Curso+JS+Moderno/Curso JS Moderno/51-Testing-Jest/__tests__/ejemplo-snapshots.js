
const usuario = {
    nombre: 'Oliver Saraguro Remache',
    saldo: 1000,
    carrera: 'Ingeniero en Informatica'
}

describe('Testisng al usuario', () => {
    test('Es Oliver Saraguro', () => {
        expect(usuario).toMatchSnapshot()
    })
})