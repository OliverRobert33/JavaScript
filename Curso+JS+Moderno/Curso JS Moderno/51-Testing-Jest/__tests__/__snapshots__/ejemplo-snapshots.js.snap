// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testisng al usuario Es Oliver Saraguro 1`] = `
{
  "carrera": "Ingeniero en Informatica",
  "nombre": "Oliver Saraguro Remache",
  "saldo": 1000,
}
`;
